### ISSUE TITLE
new installation error: unexpected keyword argument 'warn'

### ISSUE THREAD

**Issue by cfgoldsmith** [ReactionMechanismGenerator/RMG-Py#2099]
Hello. I tried a fresh installation on a new-ish iMac running Catalina (10.15.7). I followed all the instructions. When it came time to run the minimal example file, I got this error:

```
(rmg_env) franklin@Claudes-iMac-2 RMG % rmg.py input.py                          
WARNING:root:No normalization for BCUT2D_MWHI
WARNING:root:No normalization for BCUT2D_MWLOW
WARNING:root:No normalization for BCUT2D_CHGHI
WARNING:root:No normalization for BCUT2D_CHGLO
WARNING:root:No normalization for BCUT2D_LOGPHI
WARNING:root:No normalization for BCUT2D_LOGPLOW
WARNING:root:No normalization for BCUT2D_MRHI
WARNING:root:No normalization for BCUT2D_MRLOW
Traceback (most recent call last):
  File "/Users/franklin/anaconda3/envs/rmg_env/bin/rmg.py", line 48, in <module>
    from rmgpy.rmg.main import RMG, initialize_log, process_profile_stats, make_profile_graph
  File "/Users/franklin/anaconda3/envs/rmg_env/lib/python3.7/site-packages/rmgpy/rmg/main.py", line 68, in <module>
    from rmgpy.rmg.listener import SimulationProfileWriter, SimulationProfilePlotter
  File "/Users/franklin/anaconda3/envs/rmg_env/lib/python3.7/site-packages/rmgpy/rmg/listener.py", line 34, in <module>
    from rmgpy.tools.plot import SimulationPlot
  File "/Users/franklin/anaconda3/envs/rmg_env/lib/python3.7/site-packages/rmgpy/tools/plot.py", line 37, in <module>
    mpl.use('Agg', warn=False, force=False)
TypeError: use() got an unexpected keyword argument 'warn'
```

I think that the initial warning messages are perhaps related to coolprop? 
Thank you.

**Comment by mjohnson541** [comment]
This is an issue with matplotlib version: #2011. You either need to downgrade matplotlib below 3.3 or do a developer's install and use master.

**Comment by rwest** [comment]
Recommended course of action for other people finding this issue: follow the instructions for "For Developers: Installation by Source Using Anaconda Environment"  (at least until we get a new binary release, but even after that the developer version is usually better).

**Comment by mjohnson541** [comment]
With the release of v3.1.0 which contains the fix from #2011 this should be resolved.
